.golangci.yml

```diff
@@ -14,6 +14,12 @@ issues:
     # In this case the socket FDs and environment vars may be different, so we can't re-exec.
     - path: lxd/api_cluster.go
       text: "deep-exit"
+    # All lxc commands are defined against unexported structs. Each struct has Command() and Run(...)
+    # methods following the same pattern. These are self-explanatory and do not need comments. This
+    # exclude rule circumvents the "checkPrivateReceivers" argument of the "exported" rule from the
+    # "revive" linter in the lxc directory only.
+    - path: lxc
+      text: "should have comment or be unexported"
 linters-settings:
```

Comment:
@tomponline I've just seen the thread in #13504. Shall I refactor the CLI to unexport all of these methods in a separate PR?

Reply:
What is involved with doing that? Is it a large change?

Reply:
I don't think so. Maybe an hour or so

Reply:
It probably depends on any other linting errors I find in the lxc package as I'll be touching a lot of files.

Reply:
OK please proceed. It would be better for our code to comply with the linter rather than having to put ignore rules in place that may mask other issues in the future.